Content.Shared/Species/SpeciesPrototype.cs

```diff
@@ -51,4 +51,5 @@ public enum SpeciesSkinColor
 {
     HumanToned,
     Hues,
+    TintedHues,
```

Comment:
i assume this is so you can have them all, like, be white, but still have different saturation/lightness? if so, then it should be docced